components/base/kinematicbase/differentialDrive.go

```diff
@@ -90,7 +89,8 @@ func (ddk *differentialDriveKinematics) CurrentInputs(ctx context.Context) ([]re
 		return nil, err
 	}
 	pt := pif.Pose().Point()
-	theta := math.Mod(pif.Pose().Orientation().OrientationVectorRadians().Theta, 2*math.Pi) - math.Pi
+	theta := math.Mod(pif.Pose().Orientation().EulerAngles().Yaw, 2*math.Pi) - math.Pi
```

Comment:
Why this change?

Reply:
Orientation Vectors were exhibiting numerical instability

Reply:
Euler angles will have gimbal lock issues that will also show numeric stability; what you want for the Yaw value may actually be in Roll.

The proper way to do this will be to Transform an OV of OY=1 by this orientation, then take the atan2 of the OX and OY values of the resulting OV.

Reply:
They will not have gimbal lock because of the nature of the orientations in question means they will only ever rotate about yaw and therefore are we do not see changes in the domain.  On the contrary Orientation Vectors have a problem where they will dance between the two possible solutions and cause wildly different angles.

Reply:
Not a big fan of euler angles. Agree we won't see gimbal lock during normal operation, but this will break if a vehicle ever points straight up or down. But agree it's better than what currently exists. Can we add a TODO to make this more robust?